homeassistant/components/zha/core/cluster_handlers/closures.py

```diff
@@ -124,11 +124,15 @@ class WindowCoveringClient(ClientClusterHandler):
 class WindowCovering(ClusterHandler):
     """Window cluster handler."""
 
-    _value_attribute = 8
+    _value_attribute_lift = 8
+    _value_attribute_tilt = 9
```

Comment:
To avoid hardcoding magic numbers, these attributes are accessible as:

```Python
from zigpy.zcl.clusters.closures import WindowCovering

WindowCovering.AttributeDefs.current_position_lift_percentage.id
WindowCovering.AttributeDefs.current_position_tilt_percentage.id
```